src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -130,13 +131,14 @@ private static bool IsInvalidDouble(double number)
             where T : FormulaValue
         {
             return StandardErrorHandling<T>(expandArguments, replaceBlankValues, checkRuntimeTypes, checkRuntimeValues, returnBehavior, (runner, symbolContext, irContext, args) =>
-            {
-                return targetFunction(irContext, args);
+            {                
+                var result = targetFunction(irContext, args);
```

Comment:
So we are enforcing the fact that functions which do not accept the EvalVisitor/Runner must always be sync.